app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -185,10 +204,50 @@ protected static void setUpFirebase() throws ExecutionException, InterruptedExce
 
         createHouseholds();
 
-        signInTestUserWithCredentials(TEST_USERS_EMAILS[0], TEST_USERS_PWD[0]);
-
         createTestTaskList();
 
+        signInTestUserWithCredentials(TEST_USERS_EMAILS[0], TEST_USERS_PWD[0]);
+
         createFirebaseDoneFlag();
     }
+
+    // Task list loading
+    private static com.google.android.gms.tasks.Task<DocumentReference> storeTask(com.github.houseorganizer.houseorganizer.task.Task task, CollectionReference taskListRef) {
```

Comment:
Using the fully qualified name for all the task objects makes the code less readable, maybe we should consider changing the name of our Task object

Reply:
I completely agree with you, perhaps something like "HTask" could work